package.json

```diff
@@ -9,7 +9,7 @@
     "doctrine": "^0.5.2"
   },
   "peerDependencies": {
-    "tern": "^0.13.0"
+    "tern": ">=0.15"
```

Comment:
I'm aware, and that is what I tested against, but this is compatible with >=0.15